src/Common/src/Interop/Windows/user32/Interop.GetWindowLong.cs

```diff
@@ -9,7 +9,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32)]
+        [DllImport(Libraries.User32, EntryPoint = "GetWindowLongW")]
```

Comment:
Is this a known limitation in the ILC? It seems to be arbitrary if CSC can figure it out.